docs/DeveloperGuide.md

```diff
@@ -141,6 +169,32 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
```

Comment:
Line above this, should change AddressBook to MATHUTORING

Reply:
Done